src/site/content/en/blog/aria-poison-or-antidote/index.md

```diff
@@ -0,0 +1,283 @@
+`aria-activedescendant` at the right times, or forget to have a menubar containing the other menus.
+And what about menu item counts? Usually menu items are presented by screen readers with something
+like "item 3 of 5" so that the user knows where they are. This is generally counted automatically by
+the browser, but in some cases, and in some browser - screen reader combinations, the wrong numbers
+might be computed, and the author needs to override these numbers with `aria-posinset` and
+`aria-setsize`.
+
+And this is just menu bars. Think of how many kinds of widgets there are. Glance at the ARIA spec or
+authoring practices if you like. For each pattern, there are a dozen ways ARIA could be misused.
+ARIA relies on authors to know what they're doing. What could possibly go wrong, given that most
+authors are not screen reader users?
+
+In other words, it is 100% necessary for actual screen reader users to try ARIA widgets out before
+they're considered shippable. There's too much nuance. Ideally everything would be tried with
+several different browser - screen reader combinations, because of the numerous implementation
+quirks, in addition to a few incomplete implementations.
+
+### File bug on Chrome or on web content?
+
+For our team, which works on Chrome, we need to know when our representation of truth doesn't match
+what's provided by other browsers. When NVDA reads something different in the same web content in
+Firefox vs. Chrome, it's often our bug, since Firefox has an older, more seasoned accessibility
+mapping layer.
+
+If all the browser - screen reader combinations seem to agree, and the web content still behaves
+badly, we will need to file a bug on the web content itself, not on the browser.
+
+## Summary
+
+In summary, ARIA magic can be used to override anything and everything that the HTML says, or add to
+it. It can be used to do little fine changes to the accessibility presentation, or to create an
+entire experience. This is why ARIA is both incredibly powerful and yet dangerous in the hands of
+our friendly local web authors that don't generally use screen readers themselves. 
+
+ARIA is just a dumb truth override markup layer. When a screen reader asks what's happening, if ARIA
+exists, they get the ARIA version of the truth instead of the real underlying truth.
+
+## Addendum 1: Additional Resources
+
+### Hybrid reference with keyboard info and code examples
+
++   [W3C's ARIA Authoring Practices](https://www.w3.org/TR/wai-aria-practices-1.1/): this
+    documents the important keyboard navigation characteristics of each example and provides
+    working JS/CSS/ARIA code. The examples are focused on what works today, and do not cover mobile.
+
+## Addendum 2: What is ARIA most used for?
+
+Because ARIA can be used to replace or supplement small or large truths, it's kind of a generally
+useful thing for saying stuff that the screen reader cares about. 
```

Comment:
```suggestion
Because ARIA can replace or supplement small or large truths, generally
useful for saying stuff that the screen reader cares about. 
```